dcos/constants.py

```diff
@@ -14,13 +14,17 @@
 DCOS_CLUSTER_ATTACHED_FILE = "attached"
 """Name of the file indicating the current attached cluster"""
 
-DCOS_SUBCOMMAND_ENV_SUBDIR = 'env'
-"""In a package's directory, this is the cli contents subdirectory."""
+DCOS_GLOBAL_SUBCOMMAND_SUBDIR = 'global_subcommands'
```

Comment:
As the PR description states : `we can just treat the legacy subcommands folder as our new global one.`, why is there this new directory introduced?

Reply:
Ah my bad, this was the incorrect previous usage, that you changed!